content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/how-to-create-policies-chainguard-enforce-console/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "How to create policies in the Chainguard Enforce Console"
+type: "article"
+description: "Creating policies in the Chainguard Enforce UI"
+date: 2022-12-10T15:22:20+01:00
+lastmod: 2022-12-10T15:22:20+01:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-enforce-kubernetes"
+weight: 100
+toc: true
+---
+
+Security policies in Chainguard Enforce ensure that our development teams are deploying containers within set policies. 
+
+We can associate a policy YAML file with a given group in order to achieve our security goals. This guide will go through how to set up policies on the Chainguard Enforce user interface available to you via [console.enforce.dev](https://console.enforce.dev). You must already have an account with Chainguard to follow this guide. You can request access for **Chainguard Enforce for Kubernetes** on the [inquiry form](https://www.chainguard.dev/get-demo?utm_source=docs).
+
+To review sample policies that you may want to leverage, please check out our page on [Chainguard Enforce Policy Examples](chainguard-enforce-policy-examples). If you would like to use the CLI to work with policies, review our [`chainctl` policy docs](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainctl-docs/chainctl_policies_apply/).
+
+## Access the Chainguard Enforce Console
+
+To begin, you will need to access your Chainguard Enforce Console. This is available on the web to you at [console.enforce.dev](https://console.enforce.dev). You will need to authenticate your user. Once you are logged in and authenticated, the Chainguard Enforce Console landing page will be displayed. 
+
+![Chainguard Enforce Console landing page](console-landing-page.png)
+
+At the top of the page will be three icons you can click on: **Clusters**, **Policies**, and **Packages**. Click on **Policies** to pull up the policies you have available on Chainguard Enforce.
+
+## Inspect any existing policies
+
+If you were invited to a group, you may already have policies available. From the Policy landing page, you can review any existing policies and any existing violations of those poicies. 
+
+![Chainguard Enforce policies page](policies-page.png)
+
+These are displayed in a table by the policy's **Name**, **Violations** and **Groups**, similar to the screenshot below.
```

Comment:
I'd change the final part of this sentence to ", similar to the following screenshot"